falcon/request.py

```diff
@@ -475,6 +480,59 @@ def headers(self):
     def params(self):
         return self._params
 
+    @property
+    def param(self):
```

Comment:
IIRC, we decided to drop the 's' in order to avoid having to reconcile with the existing `params` property. However, I'm having second thoughts on this. It may be better to make a breaking change for Falcon 1.0 so that we don't carry this design cruft around for the foreseeable future. I suppose ParamProxy could implement the [mapping interface][1] to reduce the impact of this change (as well as a method to convert to a `dict`).

Thoughts?

[1]:https://docs.python.org/3.1/glossary.html#term-mapping